src/content/guides/getting-started.md

```diff
@@ -208,7 +208,7 @@ Open `index.html` in your browser and, if everything went right, you should see
 
 ## Modules
 
-The [`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and [`export`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) statements have been standardized in [ES2015](https://babeljs.io/learn-es2015/). Although they are not supported in most browsers yet, webpack does support them out of the box.
+The [`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and [`export`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) statements have been standardized in [ES2015](https://babeljs.io/learn-es2015/) and [are supported in most browsers](https://caniuse.com/#search=modules). Some older browsers still lag behind but Webpack supports them out of the box.
```

Comment:
`webpack` should be in lowercase. 
Suggest to change word `them` in last sentence into `modules`, because right now the context tells me that `them` is talking about `Some older browsers`